### PR TITLE
Use default sensor value for missing variables

### PR DESCRIPTION
**Pull request by caseycesari**
## Overview

If a sensor has live API data available, but is not reporting all of the variables needed for the app, use the default values for the missing variable.

Connects #106 

### Demo

`develop`:
image

This branch:
image

## Testing Instructions

- Visit the sensor overview pane for the Tinicum sensor on the deploy preview for this branch and `develop`.
- Verify that the turbidity value is set on this branch with the default sensor value.

### CODE REVIEW

File: src/app/src/sensorUtils.js [hunk 1]
```diff
@@ -72,6 +73,12 @@ export function parseRiverGaugeApiData(id, data) {
                     [variable]: Number(sensorReading.value),
                     timestamp: new Date(sensorReading.dateTime),
                 });
+            } else {
+                // If sensor data is missing for a particular variable,
+                // use the default variable value for the sensor
+                return Object.assign(acc, {
+                    [variable]: DEFAULT_SENSOR_DATA[id][variable],
+                });
```
Comment (fungjj92) [inline src/app/src/sensorUtils.js]: There's opportunity in the below function `parseRiverGaugeCsvData` line 105 to use default values. It looks like we're defaulting to 0.

Comment (caseycesari) [reply src/app/src/sensorUtils.js]: Good catch. Updated.

**Comment by caseycesari** [comment]
Fixed the failing test. It was broken on `develop`.

**Comment by caseycesari** [comment]
Thanks!

**Pull request merged**